packages/core/src/components/text-field/_text-field.scss

```diff
@@ -123,12 +123,12 @@
 
       .#{$ray-class-prefix}text-field__label,
       .#{$ray-class-prefix}text-area__label {
-        padding-top: $ray-field-v-spacing-compact;
+        padding-block-start: $ray-field-v-spacing-compact;
```

Comment:
this isn't supported in IE or Edge:

![Screen Shot 2019-05-22 at 7 01 12 PM](https://user-images.githubusercontent.com/19141291/58214530-0fade680-7cc4-11e9-80ca-99d6cbda8de1.png)

Reply:
It is however supported in the edge canary ;-)

Reply:
Does `padding-top` have an impact on right-to-left languages?

Reply:
Ah my bad, this is a remnant from my first attempt to do what was needed using logical properties. The future "correct" / pedantic way to do everything in future once logical properties are standardized (they're still draft right now) will be to drop all ideas of top / bottom / left / right in favor of block-start / block-end / inline-start / inline-end.

Anyway, it doesn't impact just supporting RTL. I'll revert this.

Reply:
TIL!